server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -41,11 +44,46 @@ export function isSystemMessage(message: ISequencedDocumentMessage) {
     }
 }
 
+export interface IProtocolHandler {
+    readonly quorum: IQuorum;
+    readonly attributes: IDocumentAttributes;
+
+    setConnectionState(connected: boolean, clientId: string | undefined);
+    snapshot(): IQuorumSnapshot;
+
+    close(): void;
```

Comment:
should this be dispose()?

Reply:
Not sure. The stack where this is used goes with the `close` school of thought, so I left it as is, mostly for maintaining consistency. I believe we talked about this before, in the `Container` context with dispose vs close but I don't think we should do this with this issue..